File: languages/php/Bloock/Entity/Identity/NumberAttribute.php
<?php

namespace Bloock\Entity\Identity;

class NumberAttribute extends Attribute
{
    public static function fromProto(\Bloock\NumberAttribute $res): NumberAttribute
    {
        return new NumberAttribute($res->getId(), $res->getValue());
    }

    public function toProto(): \Bloock\NumberAttribute
    {
        $p = new \Bloock\NumberAttribute();
        $p->setId($this->id);
        $p->setValue($this->value);
        return $p;
    }
}

File: languages/python/bloock/entity/identity/number_attribute.py
from __future__ import annotations

import bloock._bridge.proto.identity_entities_pb2 as proto
from bloock.entity.identity.attribute import Attribute


class NumberAttribute(Attribute):
    @staticmethod
    def from_proto(a: proto.NumberAttribute) -> NumberAttribute:
        return NumberAttribute(
            id=a.id,
            value=a.value,
        )

    def to_proto(self) -> proto.NumberAttribute:
        return proto.NumberAttribute(
            id=self.id,
            value=self.value,
        )
